travis-ci/vmtest/run_vmtest.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eux
+
+VMTEST_SETUPCMD="PROJECT_NAME=${PROJECT_NAME} ./${PROJECT_NAME}/travis-ci/vmtest/run_selftests.sh"
+
+echo "KERNEL: $KERNEL"
+
+# Install required packages
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main" | sudo tee -a /etc/apt/sources.list
+echo "deb http://archive.ubuntu.com/ubuntu eoan main restricted universe multiverse" | sudo tee -a /etc/apt/sources.list
+sudo apt-get -qq update
+sudo apt-get -y install dwarves=1.15-1
+sudo apt-get -qq -y install clang-10 lld-10 llvm-10
```

Comment:
Thinking about further improvements, not related to this PR:
Since we're looking into building pahole from the sources it would be great to have it built once and reused for each kernel. Same for other steps which don't depend on kernel. This would save plenty of time.

Reply:
Building pahole shouldn't take long. It's a simple and small binary, so I'm not sure it's worth doing this.